README.md

```diff
@@ -32,7 +32,7 @@ vars:
     bing_ads_database: your_schema_name
 ```
 
-For additional configurations for the source models, please visit the [Bing Ads source package](https://github.com/fivetran/dbt_bing_ads_source).
+For additional configurations for the source models, visit the [Microsoft Advertising source package](https://github.com/fivetran/dbt_microsoft_ads_source).
```

Comment:
I updated the source package link to reflect the new repo name that Kristin's implementing.